docs/source/basics/introduction.rst

```diff
@@ -0,0 +1,43 @@
+.. _mantid introduction:
+
+============
```

Comment:
This whole page has similar links to the top level index page, and might be better as a replacement of the top level index page.